src/pages/home/report/ReportActionCompose.js

```diff
@@ -289,6 +329,38 @@ class ReportActionCompose extends React.Component {
                             </>
                         )}
                     </AttachmentModal>
+                    <PopoverWithMeasuredContent
```

Comment:
Why do we need to measure the content? Can't we just make this popover some constant position on the screen? I think this can just be a regular `Popover` if you know the positioning ahead of time.

Reply:
You can use a regular `Popover` if the Emoji Picker has a constant width and height.

Reply:
Yeah, i also prefer the `PopoverWithMeasuredPosition`

Reply:
Just want to make sure these assumptions are correct so far

- The "anchor point" for this menu is always in the bottom right hand corner of the screen
- The size of the emoji menu is constant 
- All variables need to position this element are known by us prior to rendering it
- We still want to compute the positioning on the fly because it's convenient

lmk if I'm missing anything

Reply:
This can be downgraded to NAB if I'm the only one who cares about this.